pact/verify.py

```diff
@@ -46,6 +46,15 @@
 @click.option(
     'username', '--pact-broker-username',
     help='Username for Pact Broker basic authentication.')
+@click.option(
+    'broker_base_url', '--pact-broker-base-url',
+    default='',
+    help='Base URl for the Pact Broker instance to publish pacts to.')
+@click.option(
```

Comment:
This needs to be able to be specified multiple times (eg. I want to verify "master" and "prod" pacts)

Reply:
Or, allow a single `--consumer-version-tags` with multiple quotation delimited arguments, `--consumer-version-tags master prod "my feat"`

Same for the provider version tags.

Reply:
I'm still a bit confused. I've tried reducing to the ruby gem to understand how this works currently and I don't get results I expect. So:

 pact-provider-verifier --provider-base-url=http://localhost:3000 --pact-broker-base-url=http://localhost:80 --provider=provider --consumer-version-tag=dev2 --consumer-version-tag=dev

Only calls one tag in broker (been monitoring requests) and doesn't actually verify. Well as far as I can tell. If I drop the tag it verifies (or at least I see that in the output)

Reply:
I can't reproduce the multiple tag issue. You can have a play with the pact-provider-verifier docker container if that helps. 
The set up instructions are here: https://github.com/pact-foundation/pact-provider-verifier/blob/master/DEVELOPER_DOCUMENTATION.md

If you run `script/dev/broker.sh`, it will run against the test pactflow broker.

I'll try running your docker file.

Reply:
What do I do with the dockerfile to recreate the issue?